src/multiselect/__integ__/page-objects/multiselect-page.ts

```diff
@@ -15,15 +13,6 @@ export default class MultiselectPageObject extends SelectPageObject<MultiselectW
     await this.click(this.getOptionInGroup(groupNumber, optionNumber));
   }
 
-  async assertOptionSelected(groupNumber: number, optionNumber: number, isSelected: boolean) {
-    const className = await this.getElementAttribute(this.getOptionInGroup(groupNumber, optionNumber), 'class');
-    assert.equal(
-      className.indexOf(selectableItemStyles.selected),
```

Comment:
This line contained a bug, used `indexOf()` instead of `indexOf() > -1`.

Because this code is not used in any tests, it is easier to remove than fixing